src/dotnet/HQ.Server/Services/ProjectStatusReportServiceV1.cs

```diff
@@ -250,7 +259,8 @@ public async Task<Result<GetProjectStatusReportsV1.Response>> GetProjectStatusRe
                 SummaryHoursTotal = t.Status == ProjectStatus.Ongoing ? t.BookingHours : t.TotalHours,
                 SummaryHoursAvailable = t.Status == ProjectStatus.Ongoing ? t.BookingAvailableHours : t.TotalAvailableHours,
                 SummaryPercentComplete = t.Status == ProjectStatus.Ongoing ? t.BookingPercentComplete : t.TotalPercentComplete,
-                SummaryPercentCompleteSort = t.Status == ProjectStatus.Ongoing ? t.BookingPercentComplete : t.TotalPercentCompleteSort
+                SummaryPercentCompleteSort = t.Status == ProjectStatus.Ongoing ? t.BookingPercentComplete : t.TotalPercentCompleteSort,
+                IsCurrentPsrPeriod = DateOnly.FromDateTime(DateTime.Now).GetPeriodStartDate(Period.Week) == t.StartDate
```

Comment:
Use `thisWeekStart == t.StartDate` instead, using extension methods in LINQ queries can cause issues translating to SQL.